system-tests/services/test-pods.js

```diff
@@ -0,0 +1,271 @@
+              'exec': {
+                'command': {
+                  'shell': command
+                }
+              }
+            }
+          ],
+          'scaling': {
+            'kind': 'fixed',
+            'instances': 1
+          },
+          'networks': [
+            {
+              'mode': 'host'
+            }
+          ],
+          'env': {
+            'camelCase': 'test',
+            'snake_case': 'test',
+            'lowercase': 'test',
+            'UPPERCASE': 'test'
+          }
+        }]);
+
+      cy
+        .get('button')
+        .contains('Review & Run')
+        .click();
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Service ID')
+        .contains(`/${Cypress.env('TEST_UUID')}/${serviceName}`);
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Instances')
+        .contains('1');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('CPU')
+        .contains('0.1 (0.1 container-1)');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Memory')
+        .contains('10 MiB (10 MiB container-1)');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('Disk')
+        .contains('Not Supported');
+
+      cy
+        .root()
+        .configurationSection('General')
+        .configurationMapValue('GPU')
+        .contains('Not Supported');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Container Image')
+        .contains('Not Configured');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Force Pull On Launch')
+        .contains('Not Configured');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('CPUs')
+        .contains('0.1');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Memory')
+        .contains('10 MiB');
+
+      cy
+        .root()
+        .configurationSection('Containers')
+        .configurationMapValue('Command')
+        .contains(command);
+
+      cy.viewport(1000, 2000);
+
+      cy
+        .root()
+        .configurationSection('Environment Variables')
+        .then(function ($envSection) {
+          expect($envSection.get().length).to.equal(1);
+
+          const $tableRows = $envSection.find('tbody tr:visible');
+          const cellValues = [
+            ['camelCase', 'test', 'Edit'],
+            ['snake_case', 'test', 'Edit'],
+            ['lowercase', 'test', 'Edit'],
+            ['UPPERCASE', 'test', 'Edit']
+          ];
+
+          $tableRows.each(function (rowIndex) {
+            const $tableCells = cy.$(this).find('td');
+
+            expect($tableCells.length).to.equal(5);
```

Comment:
There are only 4 cells per Row.
![image](https://cloud.githubusercontent.com/assets/156010/24712146/59ccafe6-1a22-11e7-929f-31f8650db2b3.png)
![image](https://cloud.githubusercontent.com/assets/156010/24712164/62b14068-1a22-11e7-8f50-a0e709290970.png)

Tests are failing because of this.